### ISSUE TITLE
How do you even install this shit ????

### ISSUE THREAD

**Issue by startas** [DarkPacks/SevTech-Ages#3057]
Seriously, how do you even install this mod ?? I dont wanna use some shitty launcher, i just wanna play offline. I found files on curseforge website, but its zip files and doesnt get loaded as mod.

**Comment by pulli23** [comment]
You will have to use a launcher. - This is not a mod, instead it is a modpack. You will have to use a launcher to handle mod loading order and other interaction between mods.

**Comment by startas** [comment]
So is it possible to play it offline ? Do i have to create my own minecraft server ?

**Comment by startas** [comment]
Then why the freaking cheese cake there are client files downloadable on curseforge website ???

**Comment by artdude543** [comment]
Because that's how the twitch launcher works maybe? It's uses that file to download the mods and gets folders like configs etc... That's how it's always worked. The modpack side of Curse is for distribution of the zip file. It's not for you to download it's for the Twitch launcher. You got a problem with that take it to twitch.

**Comment by startas** [comment]
i only asked because the whole mods thing is a huge mess without instructions, and all these launchers are even bigger brain farts, so its impossible to understand what is what and where to even begin.

**Comment by artdude543** [comment]
Umm what? You download the Twitch launcher setup minecraft. Then find the lack hit install and your done. Like how is that hard? If my 4 year old brother can work this out I'm sure you can. The same process for Atlauncher applies this tracker is not for this help. So please refer to your local search engine for further help or even YouTube.

**Comment by artdude543** [comment]
These noobs? So the entire Twitch team? Sounds legit. They are they for a reason for third party developers and people who know what they are doing.

**Comment by startas** [comment]
Yes, so to speak, twitch is a sellout shithole, users aside. Also, as i said, these modpacks lack any kind of manual, or tutorial, so it is hard to understand what the launcher wants from you, instagram login doesnt work, and it doesnt say that it wants mojang login, and its not clear what will happen next, so for sure i will not put any real info into login fields, only to be stolen.

**Comment by artdude543** [comment]
Okay so clearly you're doing this wrong and more paranoid than my hamster.

1. https://app.twitch.tv/download Does not even use Instagram to log in. It uses Twitch or Old Curse Accounts. Then requires your Mojang account to authenticate to Minecraft. Which they even use the Launcher Mojang provide so it's technically better than ATL. But they are **both** safe...
2. https://www.atlauncher.com/downloads Is the same and does not even require a separate login. It only asks for your Mojang Login.

This is standard for modpacks and has been since they started. Your clearly a child with issues and does not want to put time and effort into reading up and/or even reading my comments. But come back with "petite" complaints about how the community has worked for yeeaaarrrssss now.

My advice finds another game. Because I don't think you'll ever overcome this barrier with yourself with not trusted a platform which is trusted by 99% of the userbase for modpacks and even some Mojang developers. I heard Staxel is a good alternative!
